solr/contrib/s3-repository/src/java/org/apache/solr/s3/S3BackupRepositoryConfig.java

```diff
@@ -40,9 +38,8 @@
     private final int proxyPort;
     private final boolean s3mock;
 
-    @SuppressWarnings({"rawtypes", "unchecked"})
-    public S3BackupRepositoryConfig(NamedList args) {
-        NamedList<String> config = args.clone();
+    public S3BackupRepositoryConfig(NamedList<?> args) {
+        NamedList<?> config = args.clone();
```

Comment:
Is this necessary?

Reply:
Nope, will remove the clone (and new variable)